src/views/text_editor.rs

```diff
@@ -510,9 +549,9 @@ impl TextEditor {
         self
     }
 
-    /// Listen for deltas applied to the editor.  
+    /// Listen for deltas applied to the editor.
```

Comment:
```suggestion
    /// Listen for deltas applied to the editor.  
```

spaces like this one are what I was referring to. Sometimes rustfmt will delete these if you have extra settings enabled

Reply:
It could be that you have a rustfmt file somewhere else on your system that is being used. this has happened to me in the past.